src/p2p/wire/messages.ts

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
Could you use these types everywhere: https://github.com/ChainSafe/lodestar/blob/master/src/types

Reply:
Can you clarify? Do you mean instead of using `buffer`, for example, use the aliased `bytes32`? And replicate that for other aliased types?

Reply:
Correct, you should use the aliased types, match them with the spec 1-1. It helps with upgrading to different versions and tracking changes